modules/installation-configuration-parameters.adoc

```diff
@@ -2109,6 +2109,17 @@ Additional GCP configuration parameters are described in the following table:
 |The name of the custom {op-system} image that the installation program is to use to boot compute machines. If you use `compute.platform.gcp.osImage.project`, this field is required.
 |String. The name of the {op-system} image.
 
+|compute:
+  platform:
+    gcp:
+      serviceAccount:
+|Specifies the email address of a {gcp-short} service account to be used during installations. This service account will be used to provision compute machines.
+[IMPORTANT]
```

Comment:
Please to remove the "IMPORTANT" section for compute.platform.gcp.serviceAccount, because the statement is not true, i.e. even if compute.platform.gcp.serviceAccount is not specified, the installer service account can be without the permissions `resourcemanager.projects.getIamPolicy` and `resourcemanager.projects.setIamPolicy`. Please refer to [my testing](https://issues.redhat.com/browse/OSDOCS-11735?focusedId=25568016&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25568016). Thanks!